packages/smooth_app/lib/pages/navigator/app_navigator.dart

```diff
@@ -326,6 +328,19 @@ enum _InternalAppRoutes {
   }
 }
 
+enum _ExternalRoutes {
+  MOBILE_APP_DOWNLOAD('/open-food-facts-mobile-app');
+
+  const _ExternalRoutes(this.path);
+
+  final String path;
+
+  @override
```

Comment:
Not really convinced by the choice of implementing `toString` here and neither by the actual implementation.

Reply:
How would you change the implementation exactly? (in general I mean)

Reply:
Oh I was just focusing on the new `enum`: I don't think we need a `toString` as the "standard implicit" `toString` will be explicit enough (something like `_ExternalRoutes.MOBILE_APP_DOWNLOAD) and there's no place we use something like `_ExternalRoutes.xxx.toString()`anyway.
I would just remove the `toString` here.

Reply:
Ok, so I've remove the two `toString` methods, and ensure we always call the `path` attribute everywhere.